tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./.ts-out",                    /* Redirect output structure to the directory. */
-    "incremental": true,
+    "incremental": false,
```

Comment:
What problem did you run into that you had to disable incremental compilation?

Reply:
The build was using cached results that did not resemble what was in the code.
The non-incremental build is still very quick and for an app as important as the Keyguard it's worth to wait the extra time to get a proper build.
The time difference is not really noticeable anyways.